dev/com.ibm.ws.opentracing.1.1/src/com/ibm/ws/opentracing/OpentracingContainerFilter.java

```diff
@@ -86,8 +86,9 @@ public void filter(ContainerRequestContext incomingRequestContext) throws IOExce
         }
 
         URI incomingUri = incomingRequestContext.getUriInfo().getRequestUri();
-        String incomingURL = incomingUri.toURL().toString();
+        String incomingURL = null;
         if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            incomingURL = incomingUri.toURL().toString();
```

Comment:
`incomingURL` can be `null` when trace is not enabled and reading `incomingURL` in line 128 will be `null.

Reply:
Yeah it should look like 1.3 and 2.0.  Joe has some clean up to do.